docs/Using-Fleet/configuration-files/README.md

```diff
@@ -264,15 +270,928 @@ spec:
+  ```
+  org_info:
+  	org_name: Fleet
+  ```
+
+#### org_info.org_logo_url
+
+The URL of the logo of the organization.
+
+- Optional setting (string).
+- Default value: none (uses Fleet's logo).
+- Config file format:
+  ```
+  org_info:
+  	org_logo_url: https://example.com/logo.png
+  ```
+
+### Server Settings
+
+#### server_settings.server_url
+
+The base URL of the fleet server, including the scheme (e.g. "https://").
+
+- Required setting (string).
+- Default value: none (provided during Fleet setup).
+- Config file format:
+  ```
+  server_settings:
+    server_url: https://fleet.example.org:8080
+  ```
+
+#### server_settings.live_query_disabled
+
+If the live query feature is disabled or not.
+
+- Optional setting (boolean).
+- Default value: `false`.
+- Config file format:
+  ```
+  server_settings:
+    live_query_disabled: true
+  ```
+
+#### server_settings.enable_analytics
+
+If sending usage analytics is enabled or not.
+
+- Optional setting (boolean).
+- Default value: `true`.
+- Config file format:
+  ```
+  server_settings:
+    enable_analytics: false
+  ```
+
+#### server_settings.debug_host_ids
+
+<!-- This section used to be named Debug host, this ensures links with the #debug-host hash still work -->
+<span id="debug-host" name="debug-host"></span>
+
+There's a lot of information coming from hosts, but it's sometimes useful to see exactly what a host is returning in order
+to debug different scenarios.
+
+So for example, let's say the hosts with ids 342 and 98 are not behaving as you expect in Fleet, you can enable verbose
+logging with the following configuration:
+
+```yaml
+---
+apiVersion: v1
+kind: config
+spec:
+  server_settings:
+    debug_host_ids:
+      - 342
+      - 98
+```
+
+Once you have collected the logs, you can easily disable the debug logging by applying the following configuration:
+
+```yaml
+---
+apiVersion: v1
+kind: config
+spec:
+  server_settings:
+    debug_host_ids: []
+```
+
+> **Warning:** This will potentially log a lot of data. Some of that data might be private. Please verify it before posting it.
+in a public channel or a GitHub issue.
+
+- Optional setting (array of integers).
+- Default value: empty.
+- Config file format:
+  ```
+  server_settings:
+    debug_host_ids:
+      - 342
+      - 98
+  ```
+
+#### server_settings.deferred_save_host
+
+Whether saving host-related information is done synchronously in the HTTP handler of the host's request, or asynchronously. This can provide better performance in deployments with many hosts. Note that this is an **experimental feature** (TODO(mna): is it?).
+
+- Optional setting (boolean).
+- Default value: `false`.
+- Config file format:
+  ```
+  server_settings:
+    deferred_save_host: true
+  ```
+
+### SMTP Settings
+
+<!-- This section replaced a section named SMTP authentication, this ensures links with the #smtp-authentication hash still work -->
+<span id="smtp-authentication" name="smtp-authentication"></span>
```

Comment:
```suggestion
```

I think we can remove this backlink.

I searched the Fleet repo and there are no links to the old section.